texworks/snapcraft.yaml

```diff
@@ -0,0 +1,40 @@
+name: texworks
+version: git
+summary: TeXworks 
+description: TeXworks is an environment for authoring TeX (LaTeX, ConTeXt, etc) documents, with a Unicode-based, TeX-aware editor, integrated PDF viewer, and a clean, simple interface accessible to casual and non-technical users.
+confinement: strict
+
+apps:
+  texworks:
+    command: qt4-launch bin/texworks
+    plugs: [home, x11, unity7]
+
+parts:
+    qt4conf:
+        build-packages: [dpkg-dev]
```

Comment:
Sorry, I got confused with the errors I was getting.
qt4-launch is needed to get the fonts loaded.
Is there another way to do that?